pennylane/devices/default_tensor.py

```diff
@@ -555,13 +547,15 @@ def _local_expectation(self, matrix, wires) -> float:
             Local expectation value of the matrix on the MPS.
         """
 
-        # We need to copy the MPS to avoid modifying the original state
-        qc = copy.deepcopy(self._circuitMPS)
+        # We need to copy the MPS since `local_expectation` modifies the state
+        qc = copy.deepcopy(self._quimb_mps)
 
         exp_val = qc.local_expectation(
             matrix,
             wires,
-            **self._expval_opts,
+            dtype=self._dtype.__name__,
+            simplify_sequence="ADCRS",
```

Comment:
Just out of curiosity: What does `ADCRS` mean here? Are there other options? Is there any reason to choose `ADCRS` here?

Reply:
When we compute expectation values using the `local_expectation` function, `quimb` uses `simplify_sequence` to determine which local tensor network simplifications to perform (and in which order).

This is one of those parameters that would be very nice to put in the `kwargs` for this device, but then there would probably be too many keyword arguments in the documentation and it would become messy. As discussed with management, in the future we can potentially find a way for the user to provide all `kwargs` that `quimb` can handle but just document some of them in PL.

[Here](https://quimb.readthedocs.io/en/latest/autoapi/quimb/tensor/tensor_core/index.html#quimb.tensor.tensor_core.TensorNetwork.full_simplify) there is the legend for how the `simplify_sequence` is handled internally in `quimb`. I performed a few tests/experiments at first and `"ADCRS"` seemed to provide the best results, but I suspect this depends on the specific structure of the circuit (num. qubits, gates, observables, and measurement type)